arbeitszeit_flask/database/repositories.py

```diff
@@ -523,6 +545,14 @@ def set_approval_date(self, approval_date: Optional[datetime]) -> Self:
             ),
         )
 
+    def set_rejection_date(self, rejection_date: Optional[datetime]) -> Self:
```

Comment:
I do not quite understand why this or the corresponding method for setting an acceptance date takes an optional datetime.  Why are these set up to allow a None datetime?